projects/lights_control/inc/lights_gpio.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+#include "event_queue.h"
+#include "status.h"
+
+typedef enum { LIGHTS_BOARD_FRONT, LIGHTS_BOARD_REAR } BoardType;
+
+StatusCode lights_gpio_init(BoardType boardtype);
+StatusCode lights_gpio_set(Event e);
```

Comment:
Events are typically passed by pointer `const Event *e` if you want a immutable version.

Reply:
good point, there's no point in not having it `const`